### ISSUE TITLE
Active slide has tabindex="-1"

### ISSUE THREAD

**Issue by ygatesoupe** [kenwheeler/slick#2341]
Active slide should be able to gain focus when tabbing with keyboard, so that its content can be rendered by screen reader (NVDA in my case).

[http://jsfiddle.net/07vdsj40/](http://jsfiddle.net/07vdsj40/)

The only way to get silde content rendered by screen reader is to click on the actual slide, which is not convenient for most screen reader users.

Active slide .slick-current should have attribute tabindex set to "0"

**Comment by ygatesoupe** [comment]
duplicate with [#2338](https://github.com/kenwheeler/slick/issues/2338)
